rmgpy/rmg/input.py

```diff
@@ -225,19 +225,22 @@ def simple_reactor(temperature,
     if not isinstance(temperature, list) and not isinstance(pressure, list) and all(
             [not isinstance(x, list) for x in initialMoleFractions.values()]):
         nSims = 1
-        # normalize mole fractions if applicable
-        total_initial_moles = sum(initialMoleFractions.values())
-        if total_initial_moles != 1:
-            logging.warning('Initial mole fractions do not sum to one; normalizing.')
-            logging.info('Original composition:')
-            for spec, molfrac in initialMoleFractions.items():
-                logging.info('{0} = {1}'.format(spec, molfrac))
-            for spec in initialMoleFractions:
-                initialMoleFractions[spec] /= total_initial_moles
-            logging.info('Normalized mole fractions:')
-            for spec, molfrac in initialMoleFractions.items():
-                logging.info('{0} = {1}'.format(spec, molfrac))
-            logging.info('')
+        # normalize mole fractions if not using a mole fraction range
+        if all([not isinstance(x, list) for x in initialMoleFractions.values()]):
```

Comment:
This should be indented to the same level as the above if statement:
```
if T,P,X are not ranges:
    nSims=1
if X are not ranges:
    normalize mole fractions
```

Reply:
My apologies, I was rushing to get to the mini TG 😬 The changes should be correct now. All commits have been squashed and rebased onto master. Thanks again for catching my mistakes!